aiohttp/web_fileresponse.py

```diff
@@ -121,30 +132,33 @@ async def _precondition_failed(
         self.content_length = 0
         return await super().prepare(request)
 
-    def _get_file_path_stat_and_gzip(
-        self, check_for_gzipped_file: bool
-    ) -> Tuple[pathlib.Path, os.stat_result, bool]:
-        """Return the file path, stat result, and gzip status.
+    def _get_file_path_stat_encoding(
+        self, accept_encoding: str
+    ) -> Tuple[pathlib.Path, os.stat_result, Optional[str]]:
+        """Return the file path, stat result, and encoding.
+
+        If an uncompressed file is returned, the encoding is set to ``None``.
```

Comment:
Looks like you lost the role from my suggestion:
```suggestion
        If an uncompressed file is returned, the encoding
        is set to :py:data:`None`.
```
This way, it can be linked to CPython docs, in the Sphinx build.